charts/osm/crds/multiclusterservice.yaml

```diff
@@ -58,6 +58,12 @@ spec:
                       name:
                         description: Name of the remote cluster
                         type: string
+                      weight:
+                        description: Load balancing weight of the remote cluster
+                        type: integer
+                        minimum: 1
```

Comment:
this doesn't seem right, the minimum should be 0

Reply:
0 means no weight at all. If that is the case, why we add that cluster?

Reply:
yes, there can be scenarios where the cluster is mentioned but there's no traffic yet